src/bno055/javaupm_bno055.i

```diff
@@ -13,6 +14,10 @@
 %ignore getLinearAcceleration(float *, float *, float *);
 %ignore getGravityVectors(float *, float *, float *);
 
+%template(floatVector) std::vector<float>;
+%template(byteVector) std::vector<uint8_t>;
+%template(intVector) std::vector<int>;
+
```

Comment:
This one also uses floats, but the stanza mentioned above is not present... Should it be?